src/Services/test/Resources/cf6.exe

```diff
@@ -1,3 +1,3 @@
 ﻿THIS IS A FAKE! 
-Tricking the FileLocatorService into thinking 
+Tricking the FileService into thinking 
```

Comment:
Now that we can mock return values from `IFileService` methods, do we need these fake resource files anymore? Couldn't we mock out `IFileService.FileExists` to return true for the relevant tests that rely on `src/Services/test/Resources/cf6.exe` and `src/Services/test/Resources/cf7.exe`?

Reply:
Turns out the tests that rely on these fake files are tests for `FileService` itself. In the past we've only ever mocked out dependencies of the class we're testing; I'm not sure what it would look like to mock out just a method of the test subject itself (in this case: `FileService.FileExists` is the method I was imagining we might mock).